views/address.tmpl

```diff
@@ -15,287 +15,302 @@
-                                        href="{{.Path}}?n={{.Limit}}&start={{add .Offset .Limit}}&txntype={{$txType}}"
-                                        data-turbolinks="false"
-                                        data-target="address.paginator"
-                                        data-action="address#nextPage"
-                                    >Next</a>
-                                </li>
-                            </ul>
-                        </nav>
-                    </div>
-                  <div data-target="address.table">
-                    {{template "addressTable" .}}
-                  </div>
-                  <div
-                      class="d-flex align-items-center justify-content-between"
-                  >
-                    {{if gt $TxnCount 0}}<a class="d-inline-block p-2 rounded download" href="/download/address/io/{{.Address}}{{if $.CRLFDownload}}?cr=true{{end}}" type="text/csv" download><span class="dcricon-download mx-1"></span> Download CSV</a>{{end}}
-                    <!-- This dummy span ensures left/right alignment of the buttons, even if one is hidden -->
-                    <span></span>
-                    <div class="d-inline-block text-right">
-                      <span class="fs14 d-block{{if not .IsMerged}} d-hide{{end}}" data-target="address.mergedMsg">
-                          *No unconfirmed transactions shown in merged views.
-                      </span>
-                      <label class="mb-0 mr-1" for="txntype">Type</label>
-                      <select
-                          name="txntype"
-                          data-target="address.txntype"
-                          data-action="change->address#changeTxType"
-                          class="form-control-sm mb-2 mr-sm-2 mb-sm-0 {{if not .Fullmode}}disabled{{end}}"
-                          {{if not .Fullmode}}disabled="disabled"{{end}}
-                      >
-                          <option {{if eq $txType "all"}}selected{{end}} value="all">All</option>
-                          <option {{if eq $txType "credit"}}selected{{end}} value="credit">Credits</option>
-                          <option {{if eq $txType "debit"}}selected{{end}} value="debit">Debits</option>
-                          <option {{if eq $txType "merged"}}selected{{end}} value="merged">Merged View</option>
-                          <option {{if eq $txType "merged_credit"}}selected{{end}} value="merged_credit">Merged Credits</option>
-                          <option {{if eq $txType "merged_debit"}}selected{{end}} value="merged_debit">Merged Debits</option>
-                      </select>
-                    </div>
-                  </div>
+      </div>
+    </div>
+    {{if not .IsDummyAddress}}
+    <div class="position-relative" data-target="address.listbox">
+        <div class="row align-items-center">
+          <div class="mr-auto mb-0 h4 col-24 col-sm-6">Transactions</div>
```

Comment:
I guess the table rows aren't actually "Transactions", but "Transaction inputs and outputs, optionally merged into blocks", which doesn't quite ring. 

It's "HIstory" on master.